packages/tools/generator-widget/README.md

```diff
@@ -13,73 +13,60 @@
 
 This generator uses the Yeoman scaffolding tool to let you quickly create a [Mendix Pluggable Widget](https://docs.mendix.com/howto/extensibility/pluggable-widgets).
 
----
-
 ## Installation
 
-First, install [Yeoman](http://yeoman.io) and @mendix/widget using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+1. Install [node.js](https://nodejs.org/) (version >= 12).
+1. Make sure that you have an [NPM](https://www.npmjs.com/package/npm) package below version 7 installed. We advise to use the latest v6 release which can be installed with:
 
-```bash
-npm install -g yo
-npm install -g @mendix/generator-widget
-```
+    ```bash
+     npm install -g npm@latest-6
+    ```
 
----
+1. Install [Yeoman](http://yeoman.io):
 
-Then generate your new project inside an empty folder:
+    ```bash
+    npm install -g yo
+    ```
 
-```bash
-yo @mendix/widget
-```
+1. Install Pluggable Widgets Generator:
 
-or automatically create the folder using:
+    ```bash
+    npm install -g @mendix/generator-widget
+    ```
 
-```bash
-yo @mendix/widget widget name
-```
+## Scaffold a widget project
```

Comment:
Nit: Similar to your comment in the docs PR: I think "scaffold a widget" and "scaffold a widget project" is basically the same thing in this context

Reply:
I am just backporting the improved docs from master. If we want to improve docs even further, let's make a new story on our backlog.